### PR TITLE
Feat/update fip0008 fip0013

### LINKED ISSUE #1259: Update specs corresponding to FIP0013
https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0013.md

### PR DESCRIPTION
**Pull request by ZenGround0**
Closes #1259 #1260 

Also fixes some out of date text

### CODE REVIEW

**Comment by ZenGround0** [comment]
Looks like either flaky or busted CI, the page builds and I see the changes locally when I run `npm start`

File: content/systems/filecoin_mining/storage_mining/_index.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ Note that the "skipped fault" allows for sector-wise fault penalties, as compare
 **Deadlines**
 
 A _deadline_ is a period of `WPoStChallengeWindow` epochs that divides a proving period.
-Sectors are assigned to a deadline on `miner.ProveCommitSector` and will remain assigned to it throughout their lifetime. Recall that Sectors are also assigned to a partition.
+Sectors are assigned to a deadline on ProveCommit, calling either `miner.ProveCommitSector`, or `miner.ProveCommitAggregate`, and will remain assigned to it throughout their lifetime. Recall that Sectors are also assigned to a partition.
```
Comment (Stebalien) [inline content/systems/filecoin_mining/storage_mining/_index.md:85]:
```suggestion
Sectors are assigned to a deadline on ProveCommit, by calling either `miner.ProveCommitSector`, or `miner.ProveCommitAggregate`, and will remain assigned to it throughout their lifetime. Recall that Sectors are also assigned to a partition.
```

(or "when calling")

File: content/systems/filecoin_mining/sector/sector-quality/_index.md [hunk 2]
```diff
@@ -57,4 +57,4 @@ The _Sector Quality Adjusted Power_ is:
 $sectorQuality = avgQuality*size$
 {{</katex>}}
 
-During `miner.PreCommitSector`, the sector quality is calculated and stored in the sector information.
+During `miner.PreCommitSector` and `miner.PreCommitSectorBatch`, the sector quality is calculated and stored in the sector information.
```
Comment (Stebalien) [inline content/systems/filecoin_mining/sector/sector-quality/_index.md:60]: Can also just say "on PreCommit".

File: content/systems/filecoin_mining/sector/adding_storage.md [hunk 3]
```diff
@@ -11,8 +11,8 @@ dashboardTests: 0
 
 A Miner adds more storage in the form of Sectors. Adding more storage is a two-step process:
 
-1. **PreCommitting a Sector**: A Miner publishes a Sector's SealedCID and makes a deposit. The Sector is now registered to the Miner, and the Miner must ProveCommit the Sector or lose their deposit.
-2. **ProveCommitting a Sector**: The Miner provides a Proof of Replication (PoRep) for the Sector. This proof must be submitted AFTER a delay (the InteractiveEpoch), and BEFORE PreCommit expiration.
+1. **PreCommitting a Sector**: A Miner publishes a Sector's SealedCID, through miner.PreCommitSector of miner.PreCommitSectorBatch, and makes a deposit. The Sector is now registered to the Miner, and the Miner must ProveCommit the Sector or lose their deposit.
+2. **ProveCommitting a Sector**: The Miner provides a Proof of Replication (PoRep) for the Sector through miner.ProveCommitSector or miner.ProveCommitAggregate. This proof must be submitted AFTER a delay (the InteractiveEpoch), and BEFORE PreCommit expiration.
```
Comment (Stebalien) [inline content/systems/filecoin_mining/sector/adding_storage.md:15]: Backtick quotes to be consistent?

**Review by Stebalien — APPROVED** [review]
Some nits. I'd just merge this without further review as it's a strict improvement.

**Review by jennijuju — APPROVED** [review]

File: content/systems/filecoin_mining/sector/adding_storage.md [hunk 4]
```diff
@@ -11,8 +11,8 @@ dashboardTests: 0
 
 A Miner adds more storage in the form of Sectors. Adding more storage is a two-step process:
 
-1. **PreCommitting a Sector**: A Miner publishes a Sector's SealedCID and makes a deposit. The Sector is now registered to the Miner, and the Miner must ProveCommit the Sector or lose their deposit.
-2. **ProveCommitting a Sector**: The Miner provides a Proof of Replication (PoRep) for the Sector. This proof must be submitted AFTER a delay (the InteractiveEpoch), and BEFORE PreCommit expiration.
+1. **PreCommitting a Sector**: A Miner publishes a Sector's SealedCID, through miner.PreCommitSector of miner.PreCommitSectorBatch, and makes a deposit. The Sector is now registered to the Miner, and the Miner must ProveCommit the Sector or lose their deposit.
```
Comment (jennijuju) [inline content/systems/filecoin_mining/sector/adding_storage.md:14]:
```suggestion
1. **PreCommitting a Sector**: A Miner publishes a Sector's SealedCID, through `miner.PreCommitSector` of `miner.PreCommitSectorBatch`, and makes a deposit. The Sector is now registered to the Miner, and the Miner must ProveCommit the Sector or lose their deposit.
```

**Review by arajasek — APPROVED** [review]

**Pull request merged**